2021/03/02 11:07:42 [emerg] 1#1: open() "/etc/nginx/nginx.conf" failed (2: No such file or directory)
2021/03/02 11:08:11 [emerg] 1#1: open() "/etc/nginx/nginx.conf" failed (2: No such file or directory)
2021/03/02 11:09:02 [emerg] 1#1: open() "/etc/nginx/mime.types" failed (2: No such file or directory) in /etc/nginx/nginx.conf:14
2021/03/03 00:43:00 [error] 21#21: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.17.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5001/", host: "0.0.0.0"
2021/03/03 00:43:23 [error] 21#21: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 172.17.0.1, server: 127.0.0.1, request: "GET /bac1 HTTP/1.1", upstream: "http://127.0.0.1:5001/bac1", host: "0.0.0.0"
2021/03/03 00:45:28 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.17.0.1, server: 127.0.0.1, request: "GET /bac1 HTTP/1.1", upstream: "http://127.0.0.1:5001/bac1", host: "0.0.0.0"
2021/03/03 00:45:43 [error] 29#29: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 172.17.0.1, server: 127.0.0.1, request: "GET /bac1 HTTP/1.1", upstream: "http://127.0.0.1:5001/bac1", host: "127.0.0.1"
2021/03/03 00:48:35 [error] 29#29: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: 127.0.0.1, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:5001/", host: "127.0.0.1"
2021/03/03 00:51:49 [emerg] 1#1: duplicate location "/" in /etc/nginx/conf.d/default.conf:13
2021/03/03 00:53:26 [error] 28#28: *9 open() "/etc/nginx/html/bac1" failed (2: No such file or directory), client: 172.17.0.1, server: 127.0.0.1, request: "GET /bac1 HTTP/1.1", host: "127.0.0.1"
2021/03/03 00:57:05 [error] 29#29: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 172.17.0.1, server: 127.0.0.1, request: "GET /api1/bac1 HTTP/1.1", upstream: "http://192.168.65.2:5001/api1/bac1", host: "127.0.0.1"
2021/03/03 01:12:13 [error] 28#28: *1 upstream prematurely closed connection while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /api2/bac2 HTTP/1.1", upstream: "http://192.168.65.2:5002/api2/bac2", host: "127.0.0.1"
2021/03/03 01:30:14 [error] 28#28: *3 upstream prematurely closed connection while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /api2/bac2 HTTP/1.1", upstream: "http://192.168.65.2:5002/api2/bac2", host: "127.0.0.1"
2021/03/03 09:32:09 [emerg] 1#1: host not found in upstream "api2" in /etc/nginx/conf.d/default.conf:14
2021/03/03 09:34:03 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /api2/bac2 HTTP/1.1", upstream: "http://172.18.0.3:80/api2/bac2", host: "127.0.0.1"
2021/03/03 09:34:11 [error] 29#29: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /api2 HTTP/1.1", upstream: "http://172.18.0.3:80/api2", host: "127.0.0.1"
2021/03/03 09:34:14 [error] 29#29: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /api2/ HTTP/1.1", upstream: "http://172.18.0.3:80/api2/", host: "127.0.0.1"
2021/03/03 09:34:41 [error] 29#29: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /api2/items/1 HTTP/1.1", upstream: "http://172.18.0.3:80/api2/items/1", host: "127.0.0.1"
2021/03/03 09:35:17 [error] 29#29: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /api2/items/1 HTTP/1.1", upstream: "http://172.18.0.3:80/api2/items/1", host: "127.0.0.1"
2021/03/03 09:37:15 [error] 28#28: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /api2/items/1 HTTP/1.1", upstream: "http://172.18.0.3:80/api2/items/1", host: "127.0.0.1"
2021/03/03 09:37:20 [error] 28#28: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /api2 HTTP/1.1", upstream: "http://172.18.0.3:80/api2", host: "127.0.0.1"
2021/03/03 09:40:07 [error] 28#28: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /api2 HTTP/1.1", upstream: "http://172.18.0.3:80/api2", host: "127.0.0.1"
2021/03/03 09:43:02 [error] 28#28: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /api2 HTTP/1.1", upstream: "http://172.18.0.3:80/api2", host: "127.0.0.1"
2021/03/03 09:59:04 [emerg] 1#1: host not found in upstream "api1" in /etc/nginx/conf.d/default.conf:7
2021/03/03 10:08:23 [error] 29#29: *3 upstream prematurely closed connection while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /api3/bac3 HTTP/1.1", upstream: "http://172.18.0.4:80/api3/bac3", host: "127.0.0.1"
2021/03/03 10:09:39 [error] 30#30: *1 upstream prematurely closed connection while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /api3/bac3 HTTP/1.1", upstream: "http://172.18.0.4:80/api3/bac3", host: "127.0.0.1"
2021/03/04 00:57:02 [error] 28#28: *1 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /api3/get_bac2 HTTP/1.1", upstream: "http://172.18.0.4:80/api3/get_bac2", host: "127.0.0.1"
2021/03/04 01:00:50 [error] 29#29: *3 upstream timed out (110: Connection timed out) while reading response header from upstream, client: 172.18.0.1, server: 127.0.0.1, request: "GET /api3/get_bac1 HTTP/1.1", upstream: "http://172.18.0.4:80/api3/get_bac1", host: "127.0.0.1"
